tests/drivers/spi/spi_loopback/boards/rv32m1_vega_ri5cy.conf

```diff
@@ -6,4 +6,3 @@
 
 CONFIG_SPI_LOOPBACK_DRV_NAME="SPI_0"
 CONFIG_SPI_LOOPBACK_SLAVE_NUMBER=2
-CONFIG_DEBUG_OPTIMIZATIONS=y
```

Comment:
Should this change be part of this PR? If yes, could you please separate it into its own commit with a descriptive commit message?

Reply:
Done :-)